notifications.bs

```diff
@@ -189,6 +212,24 @@ or vibration pattern that is not otherwise accessible to the end user.
   <li><p>If <var>options</var>'s <code>sticky</code> is true, set
   <var>notification</var>'s <a>sticky preference flag</a>.
 
+  <li><p>Set <var>notification</var>'s list of <a>actions</a> to an empty list,
```

Comment:
Sorry for not spotting this earlier. Since the `<p>` is not the only child of the `<li>` it needs to appear on its own line and indented. Same for the `<ol>` (and its children).